delivery_carrier_label_default/views/report_default_label.xml

```diff
@@ -1,12 +1,14 @@
 <odoo>
 <template id="report_default_label">
-  <t t-call="report.html_container">
+  <t t-call="web.html_container">
```

Comment:
you can use `web.basic_layout` and remove `<div class="article o_report_layout_standard">`

Reply:
fixed, thanks for the tip